_posts/2017-08-01-integrating-brat-with-micro-services.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Integrating BRAT annotation tool with micro-services
+author: Arpita Sharma
+published: false
+---
+
+Named Entity Recognition(NER) is one of the key tasks in information extraction. iLabs provides services for NER and Brat is used for creating annotations for NER, adding on it has a wholesome interface. So, should we provide basic annotation tool to our users?
+
+###About iLabs API
+iLabs provides services that annotates text data. This text data includes legal documents, references, and affiliations. Now, let's take a scenario, where we want to annotate the affiliations of authors writing journals and research papers. iLabs API works in three simple steps:
+
+#####1- Post request to save the data to be annotated
+    
+    http://api.innodatalabs.com/v1/documents/input/{filename}
+
+#####2- Get request to call the affiliation tagging service on the saved data
+
+    http://api.innodatalabs.com/v1/reference/ilabs.affiliation/{filename}
+
+#####3- Get request to bring the annotated data 
+
+    http://api.innodatalabs.com/v1/documents/output/{filename}
+
+Same can be done for legal documents and reference tagging. Any application that needs automatic tagging can make these three requests.
+
+###About Brat
+Brat is an open source web based tool used to create annotations. It allows you to add desired labels to the selected text spans. Brat also provides a wholesome interface to achieve best user experience through its visualizations.
+
+<p><a><img src="../images/brat_shot1.png" height="180" width="950"></a></p>
+
+There are many advantages to use Brat, reasons mentioned here are important from iLabs perspective:
+* High-quality visualizations with wholesome user interface
+* Integrates well with other annotation services
+* Easy to configure
+* Easily embeddable inside any web page
+
+In spite of all these good things, there is a limitation as well. It has limited use of shortcuts, one cannot control Brat by using keyboard only.  
```

Comment:
To Innodata, this might be a limitation, but others may not have an issue with this